aten/src/ATen/native/native_functions.yaml

```diff
@@ -1585,6 +1585,10 @@
     CPU: prelu_backward_cpu
     CUDA: prelu_backward_cuda
 
+- func: gelu(Tensor self) -> Tensor
+  dispatch:
```

Comment:
you should add a `python_module: nn`. For example https://github.com/pytorch/pytorch/blob/6b8c1eab83100ae52c677f83786ea8dbbe2df5be/aten/src/ATen/native/native_functions.yaml#L3557-L3558

Reply:
Thanks for pointing out this.